client/app/api/course/Assessment/Submission/Answer/Programming.js

```diff
@@ -0,0 +1,60 @@
+import BaseAssessmentAPI from '../../Base';
+
+export default class ProgrammingAPI extends BaseAssessmentAPI {
+  createProgrammingFiles(answerId, submissionFields) {
```

Comment:
Can you add some documentation for this API and for `deleteProgrammingFiles`?

Reply:
👍  @zhengwei143 You can use existing APIs as references.